# $ catkin_make -DCATKIN_WHITELIST_PACKAGES="copley"

# Start_Joy
int16 rpi_host
int16 copley_motor # 采用int变量而不是bool，可能再分出第三种情况，如重启